frontend/src/metabase/dashboard/components/DashboardEmbedAction/DashboardEmbedAction.unit.spec.tsx

```diff
@@ -58,20 +61,33 @@ const setup = ({
       currentUser: createMockUser({ is_superuser: isAdmin }),
       settings: createMockSettingsState({
         "enable-public-sharing": publicLinksEnabled,
+        "enable-embedding": embeddingEnabled,
       }),
     }),
   });
 };
 
 describe("DashboardEmbedAction", () => {
   describe("which label should be rendered", () => {
-    it("should have a `Sharing` tooltip if public sharing is true", () => {
+    it("should have a `You must enable Embedding` tooltip if embedding is disabled", async () => {
```

Comment:
there are `"when the user is admin"` block for other tests, but not this one. I believe this test is only true when users are admins as well. Should you nest some of the tests under `"when the user is admin"` block or just add the admin condition to the test name if it's not applicable to other tests under this same block.

Reply:
Yeah, that's why I thought the test name should also contain `user is an admin` because this is the tooltip that only admins should see.